react/README.md

```diff
@@ -251,10 +251,10 @@ This style guide is mostly based on the standards that are currently prevalent i
     <Foo bar="bar" />
 
     // bad
-    <Foo style={{ left: "20px" }} />
+    <Foo className='bad example' />
 
     // good
-    <Foo style={{ left: '20px' }} />
+    <Foo className="good example" />
```

Comment:
these MUST remain as "style" with an object literal; there's nothing wrong with the current examples.